Documentation/gettingstarted/k3s.rst

```diff
@@ -50,6 +50,12 @@ Should you encounter any issues during the installation, please refer to the
 Please consult the Kubernetes :ref:`k8s_requirements` for information on  how
 you need to configure your Kubernetes cluster to operate with Cilium.
 
+Mount the BPF Filesystem
+========================
+On each node, run the following to mount the BPF Filesystem:
+::
+
+     minikube ssh -- sudo mount bpffs -t bpf /sys/fs/bpf
```

Comment:
Does k3s always run on minikube? Where did that part come from?

Reply:
no, it was leftover from a bad copy/paste job